content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,170 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+## Migrating to Cypress 9.3
+
+This guide details the changes and how to change your code to make full use of
+Cypress 9.3.
+[See the full changelog for 9.3](/guides/references/changelog#9-3-0).
+
+None of the changes detailed here are breaking.
+
+### Attaching Files
+
+Attaching files to input elements or dropping them over the page is available in
+Cypress 9.3. Read the [`.selectFile()` API docs](/api/commands/selectfile) for
+more information on how this works and how to use it.
```

Comment:
```suggestion
### Selecting Files to Upload

Selecting files to upload to input elements or dropping them over the page is available in
Cypress 9.3. Read the [`.selectFile()` API docs](/api/commands/selectfile) for
more information on how this works and how to use it.
```

Reply:
Slightly different tweak since I'm reworking the introduction to match the other style.